pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -152,28 +153,28 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 type PersistentVolumeRecyclerConfiguration struct {
 	// maximumRetry is number of retries the PV recycler will execute on failure to recycle
 	// PV.
-	MaximumRetry int32 `json:"maximumRetry"`
+	MaximumRetry int32
```

Comment:
why remove the tags?

Reply:
see @liggitt's and my conversation in Slack and the PR description

Reply:
the k-c-m structs had a mix of missing tags, incorrect tags, and correct tags, but it didn't really matter because those structs were not currently used for deserializing anything yet.

this tries to make it clear which structs are currently being used for serialization and which are free to be changed/corrected without regard for existing deserialization use

for k-c-m, we need to:
1. move structs to the right packages
2. pick the actual field names and structure to use, then add serialization tags (resulting in zero violations found by the verifier in the process)
4. add deserialization paths